catalog/core/catalog-core-commons/src/test/java/ddf/catalog/filter/delegate/ValidationQueryDelegateTest.java

```diff
@@ -12,161 +12,25 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
 
-package ddf.catalog.metacard.validation;
+package ddf.catalog.filter.delegate;
 
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
 import static org.mockito.Mockito.mock;
-import static ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin.VALIDATION_ERRORS;
```

Comment:
So we should be able to have
```
import static ddf.catalog.data.impl.BasicTypes.VALIDATION_ERRORS;
import static ddf.catalog.data.impl.BasicTypes.VALIDATION_WARNINGS;
```
here and not have to use `BasicType` when we use them in the code, right?